.github/workflows/html5lib-tests.yml

```diff
@@ -0,0 +1,190 @@
+name: HTML API html5lib Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      # The html5lib tests were introduced in WordPress 6.6.
+      - '[6-9].[6-9]'
+    tags:
+      - '[6-9]+.[6-9]'
+      - '[6-9]+.[6-9].[0-9]+'
+  pull_request:
+    branches:
+      - trunk
+      - '[6-9].[6-9]'
```

Comment:
Ah, nice catch! I was thinking too limited when writing the globs.

Something like this could work, if a little verbose:
```yaml
'6.[6-9]' # 6.6+
'6.[6-9].[0-9]+' # 6.6 minors
'[7-9].[0-9]' # 7.0+
'[7-9].[0-9].[0-9]+' # 7.0+ minors
'1[0-9].[0-9]' # 10.0+
'1[0-9].[0-9].[0-9]+' # 10.0+ minors
```

Reply:
That should get us through the next few years 🙂 

On the >10 versions we could also start the first segment `[1-9][0-9]*` to cover us into far future:
`[1-9][0-9]*.[0-9]` and `[1-9][0-9]*.[0-9].[0-9]+`

Reply:
Updated in [1bd3a1b](https://github.com/WordPress/wordpress-develop/pull/6546/commits/1bd3a1bdbe6a4574608ede5e143119be89f541c0).

Can you take a quick look to confirm? 🙂

Reply:
It looks correct to me 👍 

I'm not familiar with the branching and tagging strategy, based on the globs I'm understand branches are created for versions like `X.Y`, and tags are created like `X.Y.Z`.

Reply:
That's right yeah 🙂